napari_animation/_tests/test_enum_compat.py

```diff
@@ -0,0 +1,190 @@
+"""Tests for enum compatibility across Python versions."""
+
+import sys
+from functools import partial
+from unittest.mock import patch
+
+import pytest
+
+from napari_animation._enum_compat import (
+    _HAS_ENUM_MEMBER,
+    _NEEDS_ENUM_MEMBER,
+    wrap_enum_member,
+)
+from napari_animation.easing import Easing
+from napari_animation.interpolation.interpolation_constants import (
+    Interpolation,
+)
+
+
+def test_enum_member_detection():
+    """Test that enum.member availability is correctly detected."""
+    if sys.version_info >= (3, 11):
+        assert _HAS_ENUM_MEMBER is True
+    else:
+        assert _HAS_ENUM_MEMBER is False
+
+    if sys.version_info >= (3, 13):
+        assert _NEEDS_ENUM_MEMBER is True
+    else:
+        assert _NEEDS_ENUM_MEMBER is False
+
+
+def test_wrap_enum_member_basic():
+    """Test basic wrap_enum_member functionality."""
+
+    def test_func(x):
+        return x * 2
+
+    partial_func = partial(test_func)
+    wrapped = wrap_enum_member(partial_func)
+
+    # The wrapped function should behave the same when called through enum
+    from enum import Enum
+
+    class TestEnum(Enum):
+        TEST = wrapped
+
+        def __call__(self, *args):
+            return self.value(*args)
+
+    result = TestEnum.TEST(5)
+    assert result == 10
+
+
+@pytest.mark.skip(reason="Troubleshooting")
+def test_wrap_enum_member_python_310_simulation():
```

Comment:
I don't quite get this test. We run on py310 and py313, so the other tests should cover both behaviors?